lib/eu_central_bank.rb

```diff
@@ -209,6 +221,32 @@ def update_parsed_historical_rates(doc)
     @historical_last_updated = Time.now
   end
 
+  def update_parsed_all_time_rates(raw_sdmx_doc)
+    sdmx_doc = raw_sdmx_doc.xpath("xmlns:CompactData/*[local-name()='DataSet']/*[local-name()='Series']")
+
+    store.transaction true do
+      sdmx_doc.each do |series|
+        rates = series.xpath("*[local-name()='Obs']")
```

Comment:
the reason I had to do this, is because the way the xml doc is actually not in accordance with any namespace. So the namespace ends up being in error. This is the best way I could figure out how to do it without trying to XSLT the XML.